dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OpenApiCRUDClientIT.java

```diff
@@ -127,7 +127,7 @@ public void testGA4GHBigPaging() throws IOException {
         ContainersApi containersApi = new ContainersApi(getWebClient(ADMIN_USERNAME, testingPostgres));
 
         // cleanup to make math easier
-        final List<DockstoreTool> dockstoreTools = containersApi.allPublishedContainers("0", 100, null, null, null);
```

Comment:
i.e. what happens if you copy+modify one of these tests and send up a null for example